config/config.go

```diff
@@ -23,6 +25,8 @@ type Config struct {
 	InsecureRegistries       []string                 `json:"insecureRegistries"`
 	CredentialsStore         string                   `json:"credsStore,omitempty"`
 	CredentialHelpers        map[string]string        `json:"credHelpers,omitempty"`
+	// EnvelopeType defines the envelope type for signing
+	EnvelopeType string `json:"envelopeType,omitempty"`
```

Comment:
This change is out of scope for https://github.com/notaryproject/notation/issues/203, please remove it from this PR.

Reply:
removed.